README.md

```diff
@@ -1 +1,95 @@
-# javascript-capstone-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+# HOLLYWOOD CENTRAL, a JavaScript capstone project 
+
+> It's a responsive web application based on externals APIs, which randomly displays many TV Shows, Actors and Actresses allowing the user to request details of each one through pop-up windows and give them "likes" and comments
+
+## Live Demo
+
+https://lucascomamala.github.io/javascript-capstone-project/
+
+## Video preview
+
+https://drive.google.com/file/d/1oL_YLiU6AhRZP4Yt-RLW8nrFCt7ifmbL/view?usp=sharing
```

Comment:
- [ ] It would be better if students highlights the piece of code they find interesting or is proud of making. Let us kindly address that
![Screenshot from 2022-09-03 03-41-43](https://user-images.githubusercontent.com/46388113/188254846-47062fd2-b79c-475d-8163-168cad1fe09c.png)